packages/rspack/src/Watching.ts

```diff
@@ -261,20 +262,15 @@ export class Watching {
 			this.compiler.contextTimestamps = contextTimeInfoEntries;
 		}
 
-		const modifiedFiles = (this.compiler.modifiedFiles =
-			this.#collectedChangedFiles);
-		const deleteFiles = (this.compiler.removedFiles =
-			this.#collectedRemovedFiles);
+		this.compiler.modifiedFiles = this.#collectedChangedFiles;
```

Comment:
Should we remove `this.compiler.modifiedFiles = this.#collectedChangedFiles`? I guess this is still required for `compiler.watch`